app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsViewHolder.kt

```diff
@@ -0,0 +1,53 @@
+package ar.com.wolox.android.example.ui.home.news
+
+import android.content.Context
+import android.net.Uri
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.model.News
+import ar.com.wolox.android.example.utils.Dates
+import com.bumptech.glide.Glide
+
+class NewsViewHolder(inflater: LayoutInflater, parent: ViewGroup) :
```

Comment:
This is ok. We usually define the ViewHolder in the same file as the adapter. Since it's the only place where it is going to be used. So I suggest you to move it to that file. Kotlin lets you have multiple class declaration in the same file